sys-devel/llvm/llvm-3.8.1-r1.ebuild

```diff
@@ -169,6 +169,12 @@ src_prepare() {
 	# https://bugs.gentoo.org/show_bug.cgi?id=578392
 	eapply "${FILESDIR}"/llvm-3.8-soversion.patch
 
+	# support building llvm against musl-libc
+	use elibc_musl && eapply "${FILESDIR}"/llvm-3.8-musl-fixes.patch
```

Comment:
As check that elibc_musl isn't needed, remove it in IUSE and you'll set that this patch is still applied.